### PR TITLE
PAYARA-1251 Fix empty principal after authentication in JMX

### PR DESCRIPTION
**Pull request by smillidge**
Fix empty JMX principal when MBean is accessed after authentication

### CODE REVIEW

**Comment by smillidge** [comment]
jenkins test please

**Comment by michaelranaldo** [comment]
jenkins test please

File: nucleus/admin/util/src/main/java/com/sun/enterprise/admin/util/GenericAdminAuthenticator.java [hunk 1]
```diff
@@ -37,11 +37,10 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-
+// Portions Copyright [2016] [Payara Foundation and/or it's affiliates]
```
Comment (Cousjava) [inline nucleus/admin/util/src/main/java/com/sun/enterprise/admin/util/GenericAdminAuthenticator.java]: Isn't the year 2017, not 2016?

**Comment by smillidge** [comment]
jenkins test please

**Comment by smillidge** [comment]
jenkins test please

**Pull request merged**